plugin/web/extensions/sidebar/core/components/hooks.ts

```diff
@@ -690,9 +690,13 @@ export const processOverrides = (components?: FieldComponent[], cleanseOverride?
     }
     return;
   }
+
+  const sortedComponents = [...components].sort(
+    (a, b) => (a.updatedAt?.getTime() ?? 0) - (b.updatedAt?.getTime() ?? 0),
+  );
```

Comment:
I added updatedAt property to override property with correct order.
For example, 3dtiles has transparency and building color field, and these field override `color` property on 3dtiles layer. So color property that is overridden by transparency field is always overridden by color building field, because the order of transparency field is before building color field in `DatasetComponents`.

Reply:
Doesn't the order depend on what order the user adds the field components?
Also, @lavalse and I have discussed a couple times and decided that the last component added should take priority. For example, if you have FieldComponentA that changes point color to Red, and then add FieldComponentB after that which changes point color to Blue, the point should be Blue.